src/Sulu/Bundle/LocationBundle/Geolocator/Service/NominatimGeolocator.php

```diff
@@ -25,9 +26,9 @@
 class NominatimGeolocator implements GeolocatorInterface
 {
     /**
-     * @var ClientInterface
+     * @var HttpClientInterface
```

Comment:
This still could be both `ClientInterface|HttpClientInterface` as our bc promise.